content/blog/2016/2016-07-04-ewm-alpha-version.adoc

```diff
@@ -0,0 +1,57 @@
+---
+layout: post
+title: "GSoC: External Workspace Manager Plugin alpha version"
+tags:
+- pipeline
+- plugins
+- external-workspace-manager
+- gsoc
+author: alexsomai
+---
+
+Currently it's quite difficult to share and reuse the same workspace between multiple jobs and across nodes.
+There are some possible workarounds for achieving this, but each of them has its own drawback,
+e.g. stash/unstash pre-made artifacts, Copy Artifacts plugin or advanced job settings.
+A viable solution for this problem is the External Workspace Manager plugin, which facilitates workspace share and
+reuse across multiple Jenkins jobs and nodes.
+It also eliminates the need to copy, archive or move files.
+
+I'd like to announce that an alpha version of the External Manager Plugin has been released!
+It's now public available for testing.
+To be able to install this plugin, you must follow the steps from the Experimental Plugins Update Center
+link:/blog/2013/09/23/experimental-plugins-update-center/[blog post].
```

Comment:
Would you mind adding a note stressing that the experimental update center should not be used for production installations of Jenkins? Or something like that? I want to make sure people don't use the experimental update center and then break their Jenkins and start complaining to me on Twitter :frowning: